cloudify/logs.py

```diff
@@ -371,9 +375,18 @@ def _publish_message(message, message_type, logger, skip_send=False):
     execution_id = get_execution_id()
     client = manager.get_rest_client()
     if message_type == 'log':
-        client.events.create(logs=[message], execution_id=execution_id)
+        logs = [message]
+        events = None
     else:
-        client.events.create(events=[message], execution_id=execution_id)
+        logs = None
+        events = [message]
+    client.events.create(
+        events=events,
+        logs=logs,
+        execution_id=execution_id,
+        agent_name=get_daemon_name(),
```

Comment:
Should this perhaps just be executor_name which does get_daemon_name() or get_manager_name()?

Reply:
I kinda want to go with separate, because it's just a string, it's hard to tell from just looking at it whether it's a manager or an agent.
Additionally, I envision a future in which there could be BOTH set, when you have multiple instances of a mgmtworker running on a single host.

Reply:
<thumbs up>